src/routes/ConsortiumManager/settings/users/general/PermissionSets/ConsortiumPermissionsSetForm/ConsortiumPermissionsSetForm.js

```diff
@@ -0,0 +1,107 @@
+import { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { useHistory, useLocation } from 'react-router-dom';
+import { useQueryClient } from 'react-query';
+
+import { HasCommand, Layer } from '@folio/stripes/components';
+import { useShowCallout } from '@folio/stripes-acq-components';
+import { useStripes } from '@folio/stripes/core';
+
+import { PermissionSetForm } from '../../../../../../../temp';
+
+import { TENANT_ID_SEARCH_PARAMS, PERMISSION_SET_ROUTES } from '../constants';
+
+export const ConsortiumPermissionsSetForm = ({
+  onSave,
+  onRemove,
+  initialValues,
+}) => {
+  const history = useHistory();
+  const location = useLocation();
+  const intl = useIntl();
+  const stripes = useStripes();
+
+  const tenantId = useMemo(() => new URLSearchParams(location.search).get(TENANT_ID_SEARCH_PARAMS), [location.search]);
+  const showCallout = useShowCallout();
+  const queryClient = useQueryClient();
+
+  const onCancel = () => {
+    history.push({
+      pathname: PERMISSION_SET_ROUTES.PERMISSION_SETS,
+      search: `?${TENANT_ID_SEARCH_PARAMS}=${tenantId}`,
+    });
+  };
+
+  const handleMutationSuccess = (permissionName) => {
+    onCancel();
+
+    const messageId = 'ui-consortia-settings.consortiumManager.members.permissionSets.save.permissionSet.success';
+
+    queryClient.invalidateQueries({ tenantId });
```

Comment:
Are you sure that argument (query key) should be an object?

Reply:
@usavkov-epam It is very interesting. With object, it works as expected. With string, the changes aren't reflected. I can check it.

Reply:
it should be 
{ queryKey: [namespace, tenantId, permissionId] }

Reply:
The above code supports starting from version 4. This is `([namespace, tenantId])`compatible with version 3 which we are using now.